posthog/tasks/email.py

```diff
@@ -117,6 +118,12 @@ def send_email_verification(user_id: int, token: str) -> None:
             "site_url": settings.SITE_URL,
         },
     )
+    posthoganalytics.capture(
+        user.distinct_id,
+        "verification email sent",
+        properties={"email address": user.email},
+        groups={"organization": str(user.current_organization.id)},  # type: ignore
+    )
     message.add_recipient(user.pending_email if user.pending_email is not None else user.email)
     message.send()
```

Comment:
I'd probably move the event capture until after the send just so we're certain at that point that it has been sent and there wasn't an error or something.